translations/en.json

```diff
@@ -1,3 +1,6 @@
 {
+  "Active Programs": "Active Programs",
```

Comment:
Not sure why these keep on getting removed/added

Reply:
This is the only one we are using the t function for. Perhaps related to that?